src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -71,6 +75,11 @@ public void testInitialize() {}
 	@Override
 	public void testExecute() {}
 
+	byte[] matchConfigurationCANData() {
+		return new byte[] {(byte) DriverStation.getInstance().getAlliance().ordinal(),
```

Comment:
This is definitely something to consider. Here is my thinking:
- the .ordinal() method on an enum is based on the enum values in source code order, so there isn't a chance of this changing at runtime
- the order (and hence ordinal values) could change with a new release of WPILib, but...
    - [this hasn't happened in three years](https://github.com/wpilibsuite/allwpilib/commit/8abbcf53f4cc426722a99a4ba1c0f81533d6b10e#diff-18b3c5fffeb1510ad9235ae6949be7dcR43)
    - we will be knowingly updating our WPILib versions, so this can be considered on each update. (i.e. we won't let it surprise us)